### DISCUSSION TITLE
psram wird nicht erkannt

### DISCUSSION THREAD

**Discussion by jab42** [tbnobody/OpenDTU#1873] [Q&A]
hallo,
ich habe die aktuelle version auf ein esp32cam board mit 8mb psram aufgespielt.
die infoseite zeigt mir allerdings kein psram an.
wo liegt mein fehler?
danke

**Comment by Juergen2453** [comment]
Siehe: https://github.com/tbnobody/OpenDTU/issues/1769

**Comment by jab42** [comment]
danke.
es ist zwar kein s3 sondern ein ai-thinker, aber ich werds mal testen.
sollte dort dieselbe compiler option greifen?
